sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/EpidemiologicalDataSteps.java

```diff
@@ -0,0 +1,517 @@
+        EpidemiologicalDataCasePage.ACTIVITY_SUBCONTINENT_COMBOBOX, activityData.getSubcontinent());
+    webDriverHelpers.selectFromCombobox(
+        EpidemiologicalDataCasePage.ACTIVITY_COUNTRY_COMBOBOX, activityData.getCountry());
+    //    webDriverHelpers.selectFromCombobox(
+    //        EpidemiologicalDataCasePage.ACTIVITY_STREET_INPUT, activityData.getStreet());
+    //    webDriverHelpers.selectFromCombobox(
+    //        EpidemiologicalDataCasePage.ACTIVITY_HOUSE_NUMBER_INPUT,
+    // activityData.getHouseNumber());
+    //    webDriverHelpers.selectFromCombobox(
+    //        EpidemiologicalDataCasePage.AcC_POSTAL_CODE, activityData.getPostalCode());
+    //    webDriverHelpers.selectFromCombobox(
+    //        EpidemiologicalDataCasePage.ACTIVITY_CITY_INPUT, activityData.getCity());
+    webDriverHelpers.clickOnWebElementBySelector(EpidemiologicalDataCasePage.ACTIVITY_DONE_BUTTON);
+  }
+
+  public EpidemiologicalData collectEpidemiologicalData() {
+    List<Exposure> exposures = new ArrayList<Exposure>();
+    exposures.add(collectExposureData());
+    List<Activity> activities = new ArrayList<Activity>();
+    activities.add(collectActivityData());
+
+    return EpidemiologicalData.builder()
+        .exposures(exposures)
+        .activities(activities)
+        .exposureDetailsKnown(getExposureDetailsKnown())
+        .activityDetailsKnown(getActivityDetailsKnown())
+        .residingAreaWithRisk(getResidingAreaWithRisk())
+        .largeOutbreaksArea(getLargeOutbreaksArea())
+        .contactsWithSourceCaseKnown(getContactsWithSource())
+        .build();
+  }
+
+  public YesNoUnknownOptions getActivityDetailsKnown() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.EXPOSURE_DETAILS_KNOWN_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getExposureDetailsKnown() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.EXPOSURE_DETAILS_KNOWN_OPTIONS);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getResidingAreaWithRisk() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.RESIDING_AREA_WITH_RISK);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getLargeOutbreaksArea() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.LARGE_OUTBREAKS_AREA);
+    return YesNoUnknownOptions.valueOf(Text);
+  }
+
+  public YesNoUnknownOptions getContactsWithSource() {
+
+    String Text =
+        webDriverHelpers.getCheckedOptionFromHorizontalOptionGroup(
+            EpidemiologicalDataCasePage.CONTACTS_WITH_SOURCE_CASE_KNOWN);
+    return YesNoUnknownOptions.valueOf(Text);
```

Comment:
please remove all the above get methods and use them straight in the code where they are called